cli/config/configStruct.go

```diff
@@ -16,6 +17,7 @@ type ConfigStruct struct {
 	Version                configStructs.VersionConfig `yaml:"version"`
 	View                   configStructs.ViewConfig    `yaml:"view"`
 	AgentImage             string                      `yaml:"agent-image,omitempty" readonly:""`
+	ImagePullPolicyStr     string                      `yaml:"image-pull-policy" default:"Always"`
```

Comment:
should it be "readonly"?

Reply:
i dont think it should be, then we won't be able to build and run local images